app/react/Settings/SettingsAPI.ts

```diff
@@ -20,6 +20,10 @@ export default {
       }),
     });
 
+    if (processedSettings.data?.home_page === '/') {
+      processedSettings.data.home_page = '';
+    }
```

Comment:
Is this sufficient? @RafaPolit @txau

Reply:
I think this should go on the server side rather than a client side overwrite. There is a validateSettings routine.

Reply:
I have made the update